__tests__/generator/generator-test.ts

```diff
@@ -18,7 +18,7 @@ describe('The code generator', () => {
           let output = '';
           child.stdout.on('data', (data) => { output += data; });
           child.on('close', () => {
-            fs.readFile(`${__dirname}/${name}.expected`, 'utf-8', (_err, expected) => {
+            fs.readFile(`${__dirname}/${name}.expected`, 'utf-8', (_, expected) => {
```

Comment:
Yes `_` is better, nice. Maybe even better is to promisify (for later).

Reply:
👍  good idea ! will keep in mind (for later)